weather.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Weather</title>
+    <link rel="stylesheet" href="weather.css">
+
+</head>
+<body>
+    <center>
+    <h1 id="font">Weather</h1>
+    <div class="search">
+        <input type="text" id="myText" placeholder="Search..">
+        <input type="submit" onclick="myFunction()" class="button" value="Submit">
+    </div>
+    <h2 id="demo">Beograd</h2>
```

Comment:
You dont need "Beograd" string here. Also id should be renamed to something more sensible like  " location" or "city"

Reply:
Okay changed, I put "Beograd" so every time when u opend up the site some info would be there , but I guess I can either put a string in the js file or nothing at all.